### PR TITLE
[BXMSPROD-1579] Send comment for a pull request

### PR DESCRIPTION
**Pull request by radtriste**
and setup GH format as well for markdown

https://issues.redhat.com/browse/BXMSPROD-1579

### CODE REVIEW

File: vars/pullrequest.groovy [hunk 1]
```diff
@@ -53,3 +53,22 @@ def buildSonar(String project, String settingsXmlId, String goals, String sonarC
         maven.runMavenWithSettingsSonar(settingsXmlId, goals, sonarCloudId, "${group}_${name}.maven.log")
     }
 }
+
+/**
+* This method add a comment to current PR (needs ghprb plugin running)
+*/
+void addComment(String commentText, String githubTokenCredsId = "kie-ci1-token") {
```
Comment (Ginxo) [inline vars/pullrequest.groovy:60]:
```suggestion
void postComment(String commentText, String githubTokenCredsId = "kie-ci1-token") {
```
maybe a better naming? :thinking:

File: vars/util.groovy [hunk 2]
```diff
@@ -480,13 +485,26 @@ ${additionalInfo}
 \n**Test results:**
 - PASSED: ${testResults.passCount}
 - FAILED: ${testResults.failCount}
+"""
 
+            if ('GITHUB'.equalsIgnoreCase(outputStyle)) {
```
Comment (Ginxo) [inline vars/util.groovy:490]:
```suggestion
```

File: vars/util.groovy [hunk 3]
```diff
@@ -480,13 +485,26 @@ ${additionalInfo}
...
+                summary += """
```
Comment (Ginxo) [inline vars/util.groovy:491]:
```suggestion
                summary += 'GITHUB'.equalsIgnoreCase(outputStyle) ? """
```

File: vars/util.groovy [hunk 4]
```diff
@@ -480,13 +485,26 @@ ${additionalInfo}
...
+Those are the test failures: ${failedTests.size() <= 0 ? 'none' : '\n'}${failedTests.collect { failedTest ->
+                return """<details>
+<summary>[${failedTest.fullName}](${failedTest.url})</summary>
+${failedTest.details}
+</details>"""
+}.join('\n')}
+"""   
+            } else {
```
Comment (Ginxo) [inline vars/util.groovy:499]:
```suggestion
```

File: vars/util.groovy [hunk 5]
```diff
@@ -480,13 +485,26 @@ ${additionalInfo}
...
+                summary += """
```
Comment (Ginxo) [inline vars/util.groovy:500]:
```suggestion
                :  """
```

**Review by Ginxo — APPROVED** [review]
This would be an amazing thing to have. Congrats and thanks!
just a few minor considerations...

**Review by Ginxo — COMMENTED** [review]
LGTM

**Comment by radtriste** [comment]
jenkins retest this

**Pull request merged**